seth/tts-CalculatorAssignment/JsKoo.js

```diff
@@ -0,0 +1,114 @@
+var onebtn = document.getElementById('calc-one');
+var onebtn = document.getElementById('calc-two');
+var onebtn = document.getElementById('calc-three');
+var onebtn = document.getElementById('calc-four');
+var onebtn = document.getElementById('calc-five');
+var onebtn = document.getElementById('calc-six');
+var onebtn = document.getElementById('calc-seven');
+var onebtn = document.getElementById('calc-eight');
+var onebtn = document.getElementById('calc-nine');
+var onebtn = document.getElementById('calc-zero');
+
+var decimalbtn = document.getElementById('decimal');
+var clearbtn = document.getElementById('clear');
+var backpacebtn = document.getElementById('backspace');
+var displayValElement = document.getElementById('Calculator-display');
+
+var displayVal = '0';
+var pendingVal;
+var evalStringArray = [];     /* problem here */
+
+var calcNumnbtns = document.getElementsByClassName('calc-btn-num');
+var calOperationBtns = document.getElementsByClassName('calc-btn-operator');
+
+var updateDisplayVal = (clickObj) => {
```

Comment:
The naming of your argument (`clickObj`) is kind of deceiving, since the argument that gets passed here is an event object. Typically you name it `event`. The element you clicked (or, technically the element you attached the listener to) would be the event.currentTarget (`event.target` returns whatever element was clicked, which may or may not be the element you attached the listener to. For instance if you attached it to a div, but inside the div is some other element, target may return that which may be confusing.